src/dcurl.c

```diff
@@ -33,9 +33,19 @@
 
 /* check whether dcurl is initialized */
 static bool isInitialized = false;
+static uint8_t initStatus = 0;
```

Comment:
Ditto. Use `enum` for `initStatus`.

Reply:
We might have an enum like
```
enum Status {
    C_BIT = 1,
    SSE_BIT = 1 << 1,
    AVX_BIT = 1 << 2,
    GPU_BIT = 1 << 3,
    ...
};
```
and allow the `enum` value to be assigned to the `uint8_t initStatus`?

Reply:
You can also use the statements like `1 U << 1`.

Reply:
Done.